pkgs/tools/inputmethods/ibus/wrapper.nix

```diff
@@ -7,7 +7,6 @@ let
   env = {
     buildInputs = [ ibus ] ++ plugins;
     nativeBuildInputs = [ lndir makeWrapper ];
-    propagatedUserEnvPackages = [ hicolor_icon_theme ];
```

Comment:
Is this a required package: cc @ericsagnes 
if yes adding `${hicolor_icon_theme}/share` to `XDG_DATA_DIRS` environment variable should be the appropriate fix.